app/Resources/public/less/overlay.less

```diff
@@ -22,12 +22,12 @@
             background: white;
             padding: 1em;
             height: 90%;
-            overflow: scroll;
+            overflow: auto;
```

Comment:
I remember it not showing it on mobile sometimes, that's why I set it to scroll.